install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -154,18 +154,34 @@ INSERT INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES
   ('displayAfterBodyOpeningTag', 'Very top of pages', 'Use this hook for advertisement or modals you want to load first.', '1'),
   ('displayBeforeBodyClosingTag', 'Very bottom of pages', 'Use this hook for your modals or any content you want to load at the very end.', '1');
 
-/* PHP:delete_hook(displayProductTab); */;
-/* PHP:delete_hook(displayProductTabContent); */;
-/* PHP:delete_hook(displayBeforePayment); */;
-/* PHP:delete_hook(actionBeforeAuthentication); */;
-/* PHP:delete_hook(actionOrderDetail); */;
-/* PHP:delete_hook(actionProductListOverride); */;
-/* PHP:delete_hook(actionSearch); */;
-/* PHP:delete_hook(displayCustomerIdentityForm); */;
-/* PHP:delete_hook(displayHomeTab); */;
-/* PHP:delete_hook(displayHomeTabContent); */;
-/* PHP:delete_hook(displayPayment); */;
-/* PHP:delete_hook(displayProductComparison); */;
```

Comment:
Those have been added for a reason, there's maybe a bug in the script that we should fix but deleting it is not a good idea

Reply:
is not deleted they was convert in sql queries to maintain behavior